Dockerfile

```diff
@@ -43,4 +44,4 @@ EXPOSE 26658
 EXPOSE 10997
 
 USER $USER:$USER
-ENTRYPOINT [ "burrow" ]
+ENTRYPOINT [ "burrow",  "start" ]
```

Comment:
This is a non-standard use of entrypoint, AFAIK. 

Better to either change this line to a `CMD` or to add the `start` as a `CMD` and keep the entrypoint as is.